index.html

```diff
@@ -50,10 +51,27 @@ <h1>
         </ul>
       </nav>
     </div>
+    <section class="flex-container">
+      <div id="container">
```

Comment:
Change `"container"` to a more specific label and/or use a `class` instead of `id`.